apps/dot-voting/app/store/init.js

```diff
@@ -1,17 +1,19 @@
 import { app, handleEvent } from './'
+import { initEthToken } from '../../../../shared/store-utils/token'
 
-export const initStore = () => {
+export const initStore = (network) => {
   const initialState = {
     votes: [],
     isSyncing: false,
   }
+  const settings = { network }
```

Comment:
Chad this convention of settings is utilized by other apps as a part of adopting A1's conventions for handling token balances from the vault. It's not pretty, but it will all go away when the token API is ready to go, so it didn't make sense to do a more elegant redesign when this pattern is already entrenched in other apps and utilized by A1, and on top of it, something that will remove all of this logic and relocate it to aragonJS is WIP.